pkg/services/gitops/install.go

```diff
@@ -292,3 +291,15 @@ func (g *Gitops) saveWegoConfig(ctx context.Context, params InstallParams) (*cor
 
 	return cm, nil
 }
+
+func credsFromParams(p InstallParams) *flux.HTTPSCreds {
+	var creds *flux.HTTPSCreds
```

Comment:
You could simplify this to just return nil if either string is empty, or the pointer if not.

See https://go.dev/doc/effective_go#control-structures for some guidance on this.